README.md

```diff
@@ -48,7 +47,7 @@ This repo contains 3 main folders:
   <summary> Future Meetings </summary>
 
 - 2020-08-27 Chapter 5 - Control Flow
-- 2020-09-03 Chapter 6 - Functions [Meenakshi Kushwaha](https://twitter.com/envhealthspeak)
+- 2020-09-03 Chapter 6 - Functions
```

Comment:
@meenakshi-kushwaha are you still looking to present this week? If so, @orrymr should make sure this isn't removed!

Reply:
@tanho63 Apologies, not sure how this change and the change on lines 19 & 20 of the README.md file happened! Hopefully latest commit solves these problems. My clumsiness with git surely to blame!